classes/Admin/Settings.php

```diff
@@ -441,6 +441,20 @@ public static function fields() {
 								'adblock_bypass_url_method' => 'custom',
 							),
 						),
+						'adjust_body_padding'   => array(
+							'type'    => 'checkbox',
+							'label'   => __( 'Adjust the padding added to the body when popups are shown with an overlay.', 'popup-maker' ),
```

Comment:
Since the setting only adjusting padding-right, should we say "Adjust the right padding..." or does the specific padding maybe not matter in this case to the end-user?

Reply:
Good point, semantically your right, not sure whether it matters to the user. Changing it anyways.